src/Approvals/SFA.DAS.Approvals.Api/Controllers/CohortController.cs

```diff
@@ -64,11 +64,11 @@ public async Task<IActionResult> GetCohortDetails(long cohortId)
                     return NotFound();
                 }
 
-                return Ok((GetCohortDetailsResponse)result);
+                return Ok(result);
             }
             catch (Exception e)
             {
-                _logger.LogError(e, $"Error in Get Cohort Details - cohort id {cohortId}");
+                _logger.LogError(e, $"Error in Get All Cohort Details - cohort id {cohortId}");
```

Comment:
Minor and very pedantic, but I don't quite follow the change here or what it is supposed to _mean_. The endpoint should produce all the details it needs to for the UI endpoint in context, no more, no less, so the word "All" seems redundant.

Reply:
Fixed